src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/HomeService.cs

```diff
@@ -11,20 +11,26 @@ namespace DevAdventCalendarCompetition.Services
 {
     public class HomeService : IHomeService
     {
-        private readonly IHomeRepository _homeRepository;
+        private readonly IResultsRepository _resultsRepository;
+        private readonly ITestAnswerRepository _testAnswerRepository;
+        private readonly ITestRepository _testRepository;
         private readonly IMapper _mapper;
 
         public HomeService(
-            IHomeRepository homeRepository,
+            IResultsRepository homeRepository,
```

Comment:
I think there are too many repositories here - maybe we should split it to few Services?